src/bokeh/models/plots.py

```diff
@@ -584,12 +584,10 @@ def _scale(cls, scale: Literal["auto", "linear", "log", "categorical"]) -> Scale
     outline_line_color = Override(default="#e5e5e5")
 
     renderers = List(Instance(Renderer), help="""
-    A list of all renderers for this plot, including guides and annotations
-    in addition to glyphs.
-
-    This property can be manipulated by hand, but the ``add_glyph`` and
-    ``add_layout`` methods are recommended to help make sure all necessary
-    setup is performed.
+    A list of all glyph renderers for this plot.
```

Comment:
It's not just glyph renderers, could be other renderers like graph renderers, or data renderers in general. We also could reflect those changes in the type of the property.